samples/local_attestation/common/attestation.cpp

```diff
@@ -42,7 +42,7 @@ bool Attestation::generate_local_report(
         sizeof(sha256),
         target_info_buffer,
         target_info_size,
-        report_buf,
+        &*report_buf,
```

Comment:
Why is &* needed?   that looks suspicious

Reply:
I didn't want to make a local variable to store the result.
Although thinking about more about it I'm adding one so people will be less confused.